search/mindbender-search

```diff
@@ -24,6 +24,13 @@
 ##
 # Author: Jaeho Shin <netj@cs.stanford.edu>
 # Created: 2015-07-31
+
+##Remark: the default heap size of Elastic search (1GB) may be insufficient for the application, resulting in error "Data too large, data for [type] would be larger than limit of [622775500/593.9mb]".
+# To modifiy it, run:
+# export ES_HEAP_SIZE=10g
+# before mindbender search gui. Pick a value adapted for your machine but lower than 30.5GB. More info: https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html
```

Comment:
Thanks for these notes.  Could you elaborate more on how much data you loaded when you got this error?  Also, it sounds like this heap size issue happens during search gui, but not during the update, right?

You could place this before the `##` to make it appear to the user when the `usage` is shown.